### PR TITLE
DYToLL_NNPDF31_13TeV_MllBinned***comments removed****

### PR DESCRIPTION
**Pull request by rgoldouz**

### CODE REVIEW

**Comment by efeyazgan** [comment]
Please change to:
pdfreweight 0
storeinfo_rwgt 0 ! store weight information
withnegweights 1 ! default 0

And there are still some commented inputs to be removed.

**Comment by rgoldouz** [comment]
are these three lines 
pdfreweight 0
storeinfo_rwgt 0 ! store weight information
withnegweights 1 ! default 0

have changed from 2016. because, they are copy from 2016.

Which commented lines? those which are started with ! sign?

File: bin/Powheg/production/2017/13TeV/DYToLL_NNPDF31_13TeV_MllBinned/Z_ee_NNPDF31_13TeV_M_120_200.input [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ndns1 131         ! pdf set for hadron 1 (mlm numbering)
+ndns2 131         ! pdf set for hadron 2 (mlm numbering)
+ebeam1 6500d0      ! energy of beam 1
+ebeam2 6500d0      ! energy of beam 2
+
+! To be set only if using LHA pdfs
+lhans1 306000      ! pdf set for hadron 1 (LHA numbering)
+lhans2 306000     ! pdf set for hadron 2 (LHA numbering)	
+! To be set only if using different pdf sets for the two incoming hadrons
+! QCDLambda5  0.25 ! for not equal pdf sets
```
Comment (efeyazgan) [inline bin/Powheg/production/2017/13TeV/DYToLL_NNPDF31_13TeV_MllBinned/Z_ee_NNPDF31_13TeV_M_120_200.input]: please remove his line

File: bin/Powheg/production/2017/13TeV/DYToLL_NNPDF31_13TeV_MllBinned/Z_ee_NNPDF31_13TeV_M_1400_2300.input [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ndns1 131         ! pdf set for hadron 1 (mlm numbering)
+ndns2 131         ! pdf set for hadron 2 (mlm numbering)
+ebeam1 6500d0      ! energy of beam 1
+ebeam2 6500d0      ! energy of beam 2
+
+! To be set only if using LHA pdfs
+lhans1 306000      ! pdf set for hadron 1 (LHA numbering)
+lhans2 306000     ! pdf set for hadron 2 (LHA numbering)	
+! To be set only if using different pdf sets for the two incoming hadrons
+! QCDLambda5  0.25 ! for not equal pdf sets
```
Comment (efeyazgan) [inline bin/Powheg/production/2017/13TeV/DYToLL_NNPDF31_13TeV_MllBinned/Z_ee_NNPDF31_13TeV_M_1400_2300.input]: please remove

File: bin/Powheg/production/2017/13TeV/DYToLL_NNPDF31_13TeV_MllBinned/Z_ee_NNPDF31_13TeV_M_200_400.input [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+ndns1 131         ! pdf set for hadron 1 (mlm numbering)
+ndns2 131         ! pdf set for hadron 2 (mlm numbering)
+ebeam1 6500d0      ! energy of beam 1
+ebeam2 6500d0      ! energy of beam 2
+
+! To be set only if using LHA pdfs
+lhans1 306000      ! pdf set for hadron 1 (LHA numbering)
+lhans2 306000     ! pdf set for hadron 2 (LHA numbering)	
+! To be set only if using different pdf sets for the two incoming hadrons
+! QCDLambda5  0.25 ! for not equal pdf sets
```
Comment (efeyazgan) [inline bin/Powheg/production/2017/13TeV/DYToLL_NNPDF31_13TeV_MllBinned/Z_ee_NNPDF31_13TeV_M_200_400.input]: please remove

File: bin/Powheg/production/2017/13TeV/DYToLL_NNPDF31_13TeV_MllBinned/Z_ee_NNPDF31_13TeV_M_2300_3500.input [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+ndns1 131         ! pdf set for hadron 1 (mlm numbering)
+ndns2 131         ! pdf set for hadron 2 (mlm numbering)
+ebeam1 6500d0      ! energy of beam 1
+ebeam2 6500d0      ! energy of beam 2
+
+! To be set only if using LHA pdfs
+lhans1 306000      ! pdf set for hadron 1 (LHA numbering)
+lhans2 306000     ! pdf set for hadron 2 (LHA numbering)	
+! To be set only if using different pdf sets for the two incoming hadrons
+! QCDLambda5  0.25 ! for not equal pdf sets
```
Comment (efeyazgan) [inline bin/Powheg/production/2017/13TeV/DYToLL_NNPDF31_13TeV_MllBinned/Z_ee_NNPDF31_13TeV_M_2300_3500.input]: please remove

**Comment by efeyazgan** [comment]
I commented on some of the lines to be removed. 

And yes those lines in the previous comment changed from 2016.

**Comment by rgoldouz** [comment]
Hi Efe,

I updated the files,
Can you see them?

Reza

**Comment by efeyazgan** [comment]
They look OK to me. @perrozzi,  could you please take a look as well?

**Pull request merged**